package.xml

```diff
@@ -39,6 +39,21 @@
     <notes>${notes}</notes>
     <contents>
         <dir name="/">
+            <!-- components -->
+            <!-- todo: what to do with CMakeLists.txt?
+            <file name="components/CMakeLists.txt" role="src" />
+            <file name="components/sapi/CMakeLists.txt" role="src" />
+            <file name="components/sapi/tests/CMakeLists.txt" role="test" />
+            <file name="components/string_view/CMakeLists.txt" role="src" />
+            <file name="components/string_view/tests/CMakeLists.txt" role="test" />
+            -->
```

Comment:
I'm unsure about that. The tests (which are in package.xml because they aren't the build script complains about missing files) are basically useless without them. But of course the pecl runner won't be able to do anything with them... so I don't know.

Reply:
Yeah, I don't see any value in shipping the CMake files or the Catch2 tests with PECL. I think it might be best to exclude the `components/*/tests/*.cc` files from the `verify_pecl_file_definitions` target in the Makefile. What do you think?

Reply:
Ha, I just realized that since we've backed out every attempt to add C++ files, we don't actually check that `.cc` are in PECL. This should work as long as we never add C++... which sounds good to me! (Go Rust! 🚀 )